gradle/task.go

```diff
@@ -0,0 +1,56 @@
+package gradle
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Task ...
+type Task struct {
+	name   string
+	module Module
+}
+
+// GetVariants ...
+func (task *Task) GetVariants() (Variants, error) {
+	tasksOutput, err := getGradleOutput(task.module.project.location, task.module.name+"tasks")
+	if err != nil {
+		return nil, fmt.Errorf("%s, %s", tasksOutput, err)
+	}
+
+	return cleanStringSlice(task.parseVariants(tasksOutput)), nil
+}
+
+func (task *Task) parseVariants(gradleOutput string) Variants {
+	tasks := []string{}
+lines:
+	for _, l := range strings.Split(gradleOutput, "\n") {
+		l = strings.TrimSpace(l)
+		if l == "" {
+			continue
+		}
+		l = strings.Split(l, " ")[0]
+		if strings.HasPrefix(l, task.name) {
+			for _, conflict := range conflicts[task.name] {
+				if strings.HasPrefix(l, conflict) {
+					continue lines
```

Comment:
continue lines ? what does it do?

Reply:
I used continue statement with label, which is set at line 26
https://medium.com/golangspec/labels-in-go-4ffd81932339